app/assets/javascripts/answers.js

```diff
@@ -20,12 +20,23 @@ $(document).ready(function(e){
   $("#answers").hide();
   $("#response").hide();
 
+   if($('#answer_text').val() ==  "") 
```

Comment:
There's an extra space between `==` and `""` that can be removed.
* Also, note that in JS the empty string is resolved as `false` in a boolean statement / comparison, so a shorter equivalent to this line is `if(!($('#answer_text').val()))`. However, it's fine to leave the explicit `== ""` in there, it might be a little easier to read. It's up to you!